packages/@okta/vuepress-site/guides/sign-into-mobile-app/configure-packages/android/installsdk.md

```diff
@@ -1,3 +1,11 @@
-Use the library [Okta AppAuth](https://bintray.com/okta/com.okta.android/okta-sdk-appauth-android), available through [JCenter](https://bintray.com/bintray/jcenter). To install it, add the following to your `build.grade`.
+Use the library [Okta OIDC](https://bintray.com/okta/com.okta.oidc.android/okta-oidc-android), available through [JCenter](https://bintray.com/bintray/jcenter). To install it, add the following to your `build.gradle`.
```

Comment:
Use the library [Okta OIDC](https://bintray.com/okta/com.okta.oidc.android/okta-oidc-android), available through [JCenter](https://bintray.com/bintray/jcenter). To install it, add the following to your `build.gradle`: